common/src/main/java/org/opensearch/ml/common/dataset/ImageEmbeddingInputDataSet.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.ml.common.dataset;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.FieldDefaults;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.ml.common.annotation.InputDataSet;
+
+import java.io.IOException;
+
+@Getter
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@InputDataSet(MLInputDataType.IMAGE_EMBEDDING)
+public class ImageEmbeddingInputDataSet extends MLInputDataset {
+
+    String base64Image;
```

Comment:
Do we need to support list of base64Image?

Reply:
Added support for it